webserver/templates/user/scraper.js

```diff
@@ -53,7 +53,9 @@ var Scrobble = (function() {
 
         this.scrobbledAt = function() {
             var dateContainer = rootScrobbleElement.getElementsByClassName("chartlist-timestamp")[0]
-                var dateElement = dateContainer.getElementsByTagName("span")[0];
+            if (!dateContainer)
+                return 0;
```

Comment:
Small nitpick: it's probably better to keep this on one line or add brackets around `return`.